src/sentry/api/endpoints/organization_group_index.py

```diff
@@ -0,0 +1,148 @@
+from __future__ import absolute_import, division, print_function
+
+import functools
+import six
+
+from django.conf import settings
+
+from rest_framework.response import Response
+
+from sentry.api.bases import OrganizationEventsEndpointBase
+from sentry.api.helpers.group_search import build_query_params_from_request, get_by_short_id, ValidationError
+from sentry.api.serializers import serialize
+from sentry.api.serializers.models.group import StreamGroupSerializerSnuba
+from sentry.models import Environment, Group, GroupStatus, Project
+from sentry.search.snuba.backend import SnubaSearchBackend
+
+
+ERR_INVALID_STATS_PERIOD = "Invalid stats_period. Valid choices are '', '24h', and '14d'"
+
+
+search = SnubaSearchBackend(**settings.SENTRY_SEARCH_OPTIONS)
+
+
+class OrganizationGroupIndexEndpoint(OrganizationEventsEndpointBase):
+
+    def _build_query_params_from_request(self, request, organization, project_ids):
+        projects = list(
+            Project.objects.filter(
```

Comment:
yeah sounds good, i'll do that in a follow up pr since this is already kind of a big change and i think we want to get this out so someone can start on the frontend today.